src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -58,9 +59,12 @@ struct Input
   std::string meshFile;
 
   double circleRadius {1.0};
+  std::vector<double> circleCenter {0.0, 0.0};
```

Comment:
Minor: Will this need three default values for 3D? We can add it later when we add support for 3D to the example.

Reply:
Yes.  Thanks for the catch.  I'll add a TODO note.